aopy/preproc/bmi3d.py

```diff
@@ -465,7 +465,7 @@ def find_laser_stim_times(laser_event_times, laser_event_widths, laser_event_pow
             | **corrected_times (nevent):** corrected laser timings (seconds)
             | **corrected_widths (nevent):** corrected laser widths (seconds)
             | **corrected_powers (nevent):** corrected laser powers (fraction of maximum)
-            | **times_not_found (nevent):** boolean array of times where no sensor measurement was found
+            | **times_not_found (nevent):** boolean array of times where no sensor measurement was found (either onset or offset)
```

Comment:
From reading this description it sounds like onset and offsets are different times but from the code it looks like this is a boolean array of laser pulses that are missing either an onset or offset sensor measurement.